avae/napari.py

```diff
@@ -110,7 +110,7 @@ def __init__(
         self._main_layout.addStretch(stretch=1)
         self.setMinimumWidth(400)
         self.manifold = manifold
-        self.cartestian = False  # Keeping this as false and as a placeholder as we havent implemented cartesian pose space yet
+        self.cartestian = True  # Keeping this as false and as a placeholder as we havent implemented cartesian pose space yet
```

Comment:
we should make this an option in napari, I'll add it in the fabric PR when all comes together